src/main/java/cn/nukkit/Player.java

```diff
@@ -3291,6 +3291,11 @@ public boolean chat(String message) {
         this.resetCraftingGridType();
         this.craftingType = CRAFTING_SMALL;
 
+        if(message.startsWith("/")) {
+            this.server.dispatchCommand(this, message.replaceFirst("/", ""));
```

Comment:
`message.substring(1)` would be a better solution

Reply:
I'll change that